docs/generated/openapi.yaml

```diff
@@ -2601,12 +2601,16 @@ components:
           $ref: '#/components/schemas/ProxyRule'
         toRules:
           type: array
-          description: a set of rules for the outbounds of this proxy
+          description: >-
+            a set of rules for the outbounds of this proxy. The field is not set
+            when 'meshService.mode' is 'Exclusive'.
```

Comment:
can make this description more specific? this resource does not have `meshService` param so it might be confusing